app/views/application/_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="col-lg-12">
+  <h2>New Movie</h2>
+
+  <ul>
+    <% @media.errors.each do |column, message| %>
+      <li><strong><%= column.capitalize %></strong>: <%= message %></li>
+    <% end %>
+  </ul>
+
+  <%= form_for @media do |f| %>
+    <div class="form-group">
+    <%= f.label :name %>
+    <%= f.text_field :name, class: "form-control" %>
+    <% if @media.class == Movie %>
+      <%= f.label :director %>
+      <%= f.text_field :director, class: "form-control" %>
+    <% elsif @media.class == Book %>
+      <%= f.label :author %>
+      <%= f.text_field :author, class: "form-control" %>
+    <% elsif @media.class == Album %>
+      <%= f.label :artist %>
+      <%= f.text_field :artist, class: "form-control" %>
+    <% end %>
+    <%= f.label :description %>
+    <%= f.text_area :description, class: "form-control" %>
+    <%= f.hidden_field :ranking, :value => @media.ranking %>
```

Comment:
As a general security tip, I recommend against us including the ranking information in this form. When the form is being used to create a new entry, we will always set the ranking to 0 in the controller.

When the form is being used to update an entry, we don't want the user to directly set the new ranking. However, by including that information with a hidden field (and not excluding it in the `strong_params` methods in our controllers), we've actually given the user an avenue by which to specify any value they want for ranking.